scripts/get-baseline-versions.js

```diff
@@ -0,0 +1,40 @@
+let transform = function (bbm_versions) {
+
+  var browsers = {
+    chrome: "Chrome",
+    chrome_android: "ChromeAndroid",
+    edge: "Edge",
+    firefox: "Firefox",
+    firefox_android: "FirefoxAndroid",
+    safari: "Safari",
+    safari_ios: "iOS",
+    webview_android: "Android",
+    samsunginternet_android: "Samsung",
+    opera_android: "op_mob",
+    opera: "Opera",
+    qq_android: "and_qq",
+    uc_android: "and_uc",
+  };
+
+
+  var browserslist_output = new Array();
+  bbm_versions.forEach(version => {
+    if (Object.keys(browsers).includes(version.browser)) {
+      browserslist_output.push(
+        `${browsers[version.browser]} >= ${version.version}`,
+      );
+    }
+  });
+  return browserslist_output
```

Comment:
```suggestion
  return bbm_versions
    .filter((version) => Object.keys(browsers).includes(version.browser))
		.map((version) => `${browsers[version.browser]} >= ${version.version}`);
```

Just ever so slightly more readable IMO